Backend/src/main/java/idealab/api/controller/UserController.java

```diff
@@ -29,6 +26,15 @@ public UserController(UserOperations userOperations) {
             return new ResponseEntity<>(response, HttpStatus.BAD_REQUEST);
     }
 
+    @DeleteMapping("/{id}")
+    public ResponseEntity<?> deleteUser(@PathVariable("id") Integer id) {
+        GenericResponse response = userOperations.deleteUser(id);
```

Comment:
I think it would be better to add field to GenericResponse which stores the response status. This way you don't have to put if-else into every controller which basically carries logic. I think it's better to have no logic in the controller.

`
     public class GenericResponse {
            ... 
            @JsonIgnore
             private HttpStatus responseStatus;     
    }
`

Reply:
Updated